w3lib/url.py

```diff
@@ -264,3 +264,8 @@ def any_to_uri(uri_or_path):
         return path_to_file_uri(uri_or_path)
     u = urlparse(uri_or_path)
     return uri_or_path if u.scheme else path_to_file_uri(uri_or_path)
+
+
+__all__ = ["add_or_replace_parameter", "any_to_uri", "file_uri_to_path",
```

Comment:
* `url_query_parameter`: my bad, I missed this one when looking at https://w3lib.readthedocs.org/en/latest/w3lib.html#module-w3lib.url
* `is_url` is used in scrapy even, so makes sense to add, although it's not documented.
* I'm not sure about `urljoin_rfc`, I'd prefer not, even if it's documented.

any other you had in mind?